src/main/java/uk/gov/justice/digital/delius/data/api/AppointmentCreateRequest.java

```diff
@@ -15,6 +15,14 @@
 @AllArgsConstructor
 public class AppointmentCreateRequest {
 
+    @NotNull
+    @ApiModelProperty(required = true)
```

Comment:
I dont think this should be required at this stage, pretty sure we discussed that not all appointments will be rar appointments?

Reply:
Do we need the NSI Id field too?

Thinking we will link to the NSI OR the requirement (but never the event or the offender)

Reply:
OK I will relax it the constraint here.

WRT NSI that's not a requirement for interventions (yet) so I suggest if you need it you can add it as part of your evolution of the endpoint.